docs/changelog.rst

```diff
@@ -1,5 +1,21 @@
 **Changelog**
 
+0.2.2
+^^^^^
+Monday Jul 12 2021
+
+* This version has been tested and required revisions, due to all actions used:
+
+  * ceph-osd: action `get-availability-zone`: **cs:~/rgildein/ceph-osd-0** (`LP#1911006`_)
+  * ceph-mon: action `get-health`: **cs:ceph-mon-48**
+  * ceph-mon: action `list-pools` with additional `format` argument: **cs:~/rgildein/ceph-mon-3** (`LP#1920135`_)
+  * ceph-mon: action `get-quorum-status` **not published** (`LP#1917690`_)
+  * nova-compute: action `instance-count`: **cs:~openstack-charmers-next/nova-compute-562** (`LP#1911011`_)
+  * nova-compute: action `node-name`: **cs:~openstack-charmers-next/nova-compute-562** (`LP#1911011`_)
+  * nova-compute: action `list-compute-nodes`: **cs:~openstack-charmers-next/nova-compute-562** (`LP#1911013`_)
```

Comment:
This list is a bit confusing. I think it would be more clear if the minimum revision numbers (and charm store namespaces) of the "supported" [1] charms would be shared.

A user wouldn't mind when a Juju action was introduced in the charms but which charm revision that user would need to make juju-verify work as expected.

In this case, it seems like `cs:~rgildein/ceph-osd-0`, `cs:~rgildein/ceph-mon-3` and `cs:~openstack-charmers-next/nova-compute-562` would be the "supported" [1] charms.

The nova-compute verifier section of the docs could detail which Juju actions are used by the verifier. Same for the ceph-osd and ceph-mon sections in the docs.

1. not really until they all are stable, because production environments would only use stable charms